src/main/java/InputStreamReader.java

```diff
@@ -0,0 +1,29 @@
+import org.apache.commons.io.IOUtils;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class InputStreamReader {
```

Comment:
not an indicative name- this class does more than just `read`ing an `inputstream`, it converts it to a `byte[]` as well.